### DISCUSSION TITLE
Can we create a folder inside protocol for yaml files in server project

### DISCUSSION THREAD

**Discussion by Sandeepkute** [serverpod/serverpod#558] [Q&A] [answered]
Can I create a folder structure inside the protocol folder for YAML files in my server project in serverpod project??
Like 
Authentication
-Login
-Sign up

**Comment by yahu1031** [comment, marked as answer]
@Sandeepkute Yes, you can. But there was an imports bug. It will be there out in a few days/in the next release.

**Reply by Sandeepkute** [reply]
ok thanks  @yahu1031

**Reply by yahu1031** [reply]
#330(Issue), #331(PR), #546(PR)

**Reply by yahu1031** [reply]
please mark it as answered.
